app/src/docs/_examples/tables/ArrayTable.example.tsx

```diff
@@ -21,20 +21,20 @@ export default function ArrayDataTableExample() {
             {
                 key: 'id',
                 caption: 'Id',
-                render: (item) => <Text color="gray80">{item.id}</Text>,
+                render: (item) => <Text color="brand">{item.id}</Text>,
```

Comment:
Try to use this matching for text colors:
--uui-text-brand: var(--gray90);
    --uui-text-primary: var(--gray80);
    --uui-text-secondary: var(--gray70);
    --uui-text-tertiary: var(--gray60);
    --uui-text-disabled: var(--gray50);
    --uui-text-contrast: var(--gray5);
    
It cover all text colors for promo and can be used for other themes